xcmailr-webapp/src/main/java/services/CheckDBForMailAddressDuplicates.java

```diff
@@ -68,8 +68,8 @@ private void validateMailboxAddresses() throws Throwable
     {
         final HashSet<String> mailAddresses = new HashSet<>();
         final HashMap<Long, String> idToMailAddress = new HashMap<>();
-        final PagingList<MBox> boxes = Ebean.find(MBox.class).select("id, address, domain").findPagingList(1_000);
-        for (final MBox box : boxes.getAsList())
+        final PagedList<MBox> boxes = Ebean.find(MBox.class).select("id, address, domain").setMaxRows(1_000).findPagedList();
+        for (final MBox box : boxes.getList())
```

Comment:
Please check if this construct really finds and processes all mailboxes or just the first 1,000.

Reply:
Changed to iterator approach (recommended for queries w/ large result sets).